src/main/js/helpers/axis.js

```diff
@@ -641,7 +641,16 @@ const getAxisLabelHeight = (label) => {
     const svg = dummy.append("svg");
     const grouper = svg.append("g");
     buildAxisLabel(grouper, label);
-    const height = grouper.node().getBoundingClientRect().height;
+    let height = 0;
+    if (window.navigator.platform === "Win32") {
+        if (window.navigator.userAgent.indexOf("Firefox") > -1) {
+            height = grouper.node().getBoundingClientRect().height - 3.5;
+        } else {
+            height = grouper.node().getBoundingClientRect().height - 2;
+        }
```

Comment:
Can we not introduce any browser specific code? I would like to stay away from browser/OS level code. 
If the fix can be resolved throughout the OSes and Browsers by increasing the padding then I think that is justification enough for us to do that.

Reply:
Fixed here https://github.com/cerner/carbon-graphs/pull/107/commits/19c7116da55ad4aaf689cbe1cb28abd648c1139e